client/my-sites/media-library/list-item-video.jsx

```diff
@@ -15,7 +15,8 @@ module.exports = React.createClass( {
 
 	propTypes: {
 		media: React.PropTypes.object,
-		maxImageWidth: React.PropTypes.number
+		maxImageWidth: React.PropTypes.number,
+		thumbnailType: React.PropTypes.string,
```

Comment:
Technically no, however it will be used in another PR that builds on this one - I guess it should have gone there instead, but I'll leave it here to save messing about!